crates/net/network/src/transactions/mod.rs

```diff
@@ -241,7 +244,8 @@ impl<N: NetworkPrimitives> TransactionsHandle<N> {
 pub struct TransactionsManager<
     Pool,
     N: NetworkPrimitives = EthNetworkPrimitives,
-    P: TransactionPropagationPolicy = TransactionPropagationKind,
+    PropPolicy: TransactionPropagationPolicy = TransactionPropagationKind,
+    AnnouncePolicy: AnnouncementFilteringPolicy = StrictAnnouncementFilter,
```

Comment:
the alternative to this would be to bundle these two into one singe trait acts like `NetworkworPolicies` but combines both into one trait